puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
I tried removing the shebangs, no dice. I also tried adding activation of the rvm environment, using rvm gemset use explicitly and still hitting the same issue without any insight into. If you have other ideas, I am happy to try otherwise I may have to abandon this strategy.